### PR TITLE
This is Assignment1

### PR DESCRIPTION
**Pull request by tejashrighogare**

### CODE REVIEW

File: Assignment1/SDETAssignment/src/Occurances/Occurancesassign.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package Occurances;
+//import java.util.*; 
+public class Occurancesassign{
+	
+	public static void main(String args[])
```
Comment (vrushalee123) [inline Assignment1/SDETAssignment/src/Occurances/Occurancesassign.java]: Please do code indentation

File: Assignment1/SDETAssignment/src/Occurances/Occurancesassign.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	{
```
Comment (amitsahu8) [inline Assignment1/SDETAssignment/src/Occurances/Occurancesassign.java]: need to discuss

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: Assignment1/SDETAssignment/src/Occurances/Occurancesassign.java (hunk 1 above)
Comment (tejashrighogare) [reply Assignment1/SDETAssignment/src/Occurances/Occurancesassign.java]: Yes

File: Assignment1/SDETAssignment/src/Occurances/Occurancesassign.java (hunk 2 above)
Comment (tejashrighogare) [reply Assignment1/SDETAssignment/src/Occurances/Occurancesassign.java]: Yes, sure

File: arrayJava.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package Occurances;
+
+public class arrayJava {
+
+	public static void main(String[] args) {
+		
+			 
+			int a[]=new int[5];
+			a[0]=10;
```
Comment (amitsahu8) [inline arrayJava.java]: remove extra spaces and format the code

File: arrayMultidimensional.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+	int[][] array = new int[4][4];
+	int value = 1;
+	for(int i = 0; i < 4; i++) {
+			for(int j = 0; j < 4; j++) {
+				array[i][j] = value;
+				value++;
+			}
+		}
+
+		System.out.println("The 2D array is: ");
```
Comment (amitsahu8) [inline arrayMultidimensional.java]: no need of second for loop, print the array in first for loop

File: ifElseIf.java [hunk 5]
```diff
@@ -0,0 +1,21 @@
+package Occurances;
+
+public class ifElseIf {
+
+	public static void main(String[] args) {
+		
+			String city = "Delhi";  
+			if(city == "Meerut") {  
+			System.out.println("city is meerut");
```
Comment (amitsahu8) [inline ifElseIf.java]: code format issue

File: sortColour.java [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+		}
+
+		System.out.print("Output is:");
+		printArray(array);
+	}
+
+	public static void main(String[] args) {
+
+		int[] array = new int[] { 2, 0, 2, 1, 1, 0 };
+
```
Comment (amitsahu8) [inline sortColour.java]: remove extra spaces

File: timeDifference.java [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+			time2="24:00";
+		SimpleDateFormat format = new SimpleDateFormat("HH:mm");
+		Date date1 = format.parse(time1);
+		Date date2 = format.parse(time2);
+		
+		long difference = date2.getTime() - date1.getTime(); 
+		//System.out.println(difference);
+		long diffMinutes = difference / (60 * 1000);     
+		
+		System.out.println("Time in minutes: " + diffMinutes + " minutes.");
```
Comment (amitsahu8) [inline timeDifference.java]: Print text should be "Time differences in minutes: "

File: timeDifference.java [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+import java.util.Date;
+
+public class timeDifference {
+
+	public static void main(String[] args) throws ParseException {
+		String time1 = "11:00";
+		String time2 = "12:00";
+
+		if(time2=="00:00")
+			time2="24:00";
```
Comment (amitsahu8) [inline timeDifference.java]: parameterize the code means write the code is different method and call it in main

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

**Pull request merged**